DB-3708: audit readmes and docs (#277)
pantheon-systems/decoupled-kit-js: web/docs/Packages/nextjs-kit/index.md

---
id: "index"
title: "@pantheon-systems/nextjs-kit"
sidebar_label: "Readme"
sidebar_position: 0
custom_edit_url: null
---

# Pantheon Systems Next.js Kit

> The `@pantheon-systems/nextjs-kit` is a collection of utilities and Next.js

Reviewer: ```suggestion
The `@pantheon-systems/nextjs-kit` consists of utilities and Next.js
```

> components for use in a Pantheon Decoupled Site built with Next.js.

Reviewer: Are we calling it Decoupled or Front-End site in this repo?

```suggestion
components for use in a Pantheon Front-End Site built with Next.js.
```

Author: We have a ticket to go through and clean up the names. I think we're still on the fence about where to keep Decoupled and where to switch it. I think it makes sense here, will commit this one.

## Installation

> To install this package to use in your application:

Reviewer: ```suggestion
To install this package for use in your application:
```
